src/neo/SmartContract/ApplicationEngine.Storage.cs

```diff
@@ -8,18 +8,18 @@ namespace Neo.SmartContract
 {
     partial class ApplicationEngine
     {
-        public const long StoragePrice = 100000;
+        public const long StoragePerByteRatio = 100;
```

Comment:
I think it should be decoupled from `FeePerByte` in `PolicyContract`.

Reply:
Should it be a policy value still? Or just 100000 as it was?

Reply:
Being a policy value is better. But we can do it in another PR.

Reply:
Added new policy value.